pkg/apis/core/validation/validation_test.go

```diff
@@ -19793,10 +19794,46 @@ func TestValidatePodTemplateSpecSeccomp(t *testing.T) {
 				},
 			},
 		},
+		{
+			description: "seccomp field and container annotation must not match when invalid seccomp annotations are allowed",
```

Comment:
Is "must" accurate here? Should this say "may not match" when invalid allowed?

Reply:
Sure, fixed as suggested :+1: